publication-event-handlers/src/main/java/no/unit/nva/publication/events/handlers/delete/DeleteImportCandidateEventConsumer.java

```diff
@@ -31,6 +35,10 @@ public class DeleteImportCandidateEventConsumer
     public static final int UNIQUE_HIT_FROM_SEARCH_API = 0;
     public static final String COULD_NOT_FETCH_UNIQUE_IMPORT_CANDIDATE_MESSAGE = "Could not fetch unique import "
                                                                                  + "candidate";
+    public static final String NO_IMPORT_CANDIDATE_FOUND = "No import candidate found with scopus identifier %s";
+    private static final Logger logger = LoggerFactory.getLogger(DeleteImportCandidateEventConsumer.class);
+    public static final int ZERO_HITS = 0;
```

Comment:
private

Reply:
This goes for all constant fields in a class that are not used elsewhere. FYI When creating them in Intellij they default public.

Reply:
Fixed 👍